pages/contributors.tsx

```diff
@@ -73,83 +76,286 @@ const ContributorsPage: FC<{ contributors: Contributor[] }> = ({
+                    Developer
+                  </div>
+                </div>
+                <div className="text-center">
+                  <div className="text-2xl text-gray-800 dark:text-violet-400 mt-2 mb-1">
+                    {contributor.name}
+                  </div>
+                  <div className="text-gray-400 mb-2 pb-4">Web Developer</div>
+                </div>
               </div>
-              <div className="text-gray-400 mb-2 pb-4 pt-1 m-2">
-                {contributor.contributions} Contributions
+              <div className="flex justify-between mx-6 items-center mt-4">
+                <div className="hover:bg-[#9F87FF1A] hover:text-gray-600 dark:hover:text-violet-300 py-1 px-4 ml-0 rounded-md transition-all duration-300 ease-in-out">
+                  <Link
+                    href={`https://github.com/rupali-codes/LinksHub/commits?author=${contributor.login}`}
+                    aria-label={`Commit History of ${contributor.login} in LinksHub`}
+                    className="flex flex-col items-center justify-center"
+                    {...linkProps}
+                  >
+                    <div className="pb-1 text-2xl">
+                      {contributor.contributions}
+                    </div>
+                    <span className="text-sm">Contributions</span>
+                  </Link>
+                </div>
+                <div className="mt-1 hover:bg-[#9F87FF1A] hover:text-gray-600 dark:hover:text-violet-300 py-1 px-4 mx-2 rounded-md transition-all duration-300 ease-in-out text-sm">
+                  <Link
+                    href={`https://github.com/${contributor.login}`}
+                    aria-label={`GitHub Profile of ${contributor.login}`}
+                    className="flex flex-col items-center justify-center"
+                    {...linkProps}
+                  >
+                    <div className="pb-2">
+                      <FaGithub className="text-2xl" />{' '}
+                    </div>
+                    GitHub
+                  </Link>
+                </div>
+                {contributor.twitter_username && (
+                  <div className="mt-1 hover:bg-[#9F87FF1A] hover:text-gray-600 dark:hover:text-violet-300 py-1 px-4 mr-2 rounded-md transition-all duration-300 ease-in-out text-sm">
+                    <Link
+                      href={`https://twitter.com/${contributor.twitter_username}`}
+                      aria-label={`Twitter Profile of ${contributor.twitter_username}`}
+                      className="flex flex-col items-center justify-center"
+                      {...linkProps}
+                    >
+                      <div className="pb-2">
+                        {/* <div className="bg-black rounded-full p-2"> */}
```

Comment:
Any specific reason for keeping this comment?

Reply:
Nah no reason at all. I will remove it with the next commit.